Koronavilkku/Koronavilkku/Model/ExposureModel.swift

```diff
@@ -1,14 +1,67 @@
 import Foundation
 import ExposureNotification
 
-struct Exposure: Codable {
+enum ExposureStatus: Equatable {
+    case unexposed
+    case exposed(notificationCount: Int?)
+}
+
+struct ExposureNotification: Codable {
+    static let secondsInADay = TimeInterval(60 * 60 * 24)
     
     /// How long the notification is being shown after the exposure
     ///
     /// The official exposure retention time is 10 * 24 hours from the time of exposure.
     /// As we don't know the exact time of exposure, just the 24-hour window starting
     /// from 00:00 UTC, we need to add one additional 24-hour window to make sure every
     /// possible exposure is covered.
+    static let retentionTime = secondsInADay * (10 + 1)
+    
+    /// Detection duration is 10 days long
+    static let detectionIntervalDuration = secondsInADay * 9
+    static let detectionIntervalStart = 0 - secondsInADay - detectionIntervalDuration
+
+    let detectedOn: Date
+    let expiresOn: Date
```

Comment:
Well, this is funny. Based on the exactly same reasoning I chose the opposite direction here, so that if the retention time changes in the future (or if the expiration date determination logic changes), it will still preserve any existing exposure information. Also, to fix #9.

Reply:
Having the expiration time parameterized leaves the room for dynamic retention times in the future should we decide to implement something like that